GooglePlayInstant/Editor/QuickDeploy/ErrorLogger.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using UnityEditor;
+
+namespace GooglePlayInstant.Editor.QuickDeploy
+{
+    /// <summary>
+    /// Utility class for displaying popup window errors.
+    /// </summary>
+    public static class ErrorLogger
```

Comment:
This class doesn't seem to be used anywhere? I can't find any references.

Reply:
See lines 160, 274, 314, 361 of Quick Deploy Window? Did it not show up for you?